include/ifc/file.hxx

```diff
@@ -503,9 +504,12 @@ namespace ifc {
             if (header == nullptr)
                 return false;
 
-            if (header->version > CurrentFormatVersion
-                || (header->version < MinimumFormatVersion && header->version != EDGFormatVersion))
-                throw UnsupportedFormatVersion{header->version};
+            if (!implies(options, IfcOptions::SkipVersionCheck))
+            {
+                if (header->version > CurrentFormatVersion
```

Comment:
I would have combined this as an `and`

Reply:
Let's make a later pass at unifying the boolean logic operators.  It's something that has annoyed me here, too :).